src/transformers/modeling_lxmert.py

```diff
@@ -923,79 +1055,71 @@ def _get_resized_qa_labels(self, cur_qa_logit_layer, num_labels):
             Labels for computing the masked language modeling loss.
             Indices should be in ``[-100, 0, ..., config.vocab_size]`` (see ``input_ids`` docstring)
             Tokens with indices set to ``-100`` are ignored (masked), the loss is only computed for the tokens with labels
             in ``[0, ..., config.vocab_size]``
+        obj_labels: (``Dict[Str: Tuple[Torch.FloatTensor, Torch.FloatTensor]]``, `optional`, defaults to :obj: `None`):
+            each key is named after each one of the visual losses and each element of the tuple is of the shape
+            ``(batch_size, num_features)`` and ``(batch_size, num_features, visual_feature_dim)``
+            for each the label id and the label score respectively
         matched_label (``torch.LongTensor`` of shape ``(batch_size,)``, `optional`, defaults to :obj:`None`):
             Labels for computing the whether or not the text input matches the image (classification) loss. Input should be a sequence pair (see :obj:`input_ids` docstring)
             Indices should be in ``[0, 1]``.
             ``0`` indicates that the sentence does not match the image
             ``1`` indicates that the sentence does match the image
-        input_ids (``torch.LongTensor`` of shape ``(batch_size, sequence_length)``):
-            Indices of input sequence tokens in the vocabulary.
-            To match pre-training, LXMERT input sequence should be formatted with [CLS] and [SEP] tokens as follows:
-            Indices can be obtained using :class:`transformers.LxmertTokenizer`.
-            See :func:`transformers.PreTrainedTokenizer.encode` and
-            :func:`transformers.PreTrainedTokenizer.convert_tokens_to_ids` for details.
-        visual_feats: (:obj: `Tuple[torch.FloatTensor, torch.FloatTensor]`):
-            the first item in the tuple represents the actual visual features
-            (ROI pooled object features from bounding boxes using a faster-RCNN model) of shape ՝(batch_size, num_visual_features, visual_feat_dim)՝
-            and where the second item represents the normalized bounding boxes on a scale of 0~1  of shape ՝(batch_size, 4)՝.
-            These are currently not provided by the transformers library
-        obj_labels: (``Dict[Str: Tuple[Torch.FloatTensor, Torch.FloatTensor]]``, `optional`, defaults to :obj: `None`):
-            each key is named after each one of the visual losses and each element of the tuple is of the shape
-            ``(batch_size, num_features)`` and ``(batch_size, num_features, visual_feature_dim)``
-            for each the label id and the label score respectively
         ans: (``Torch.Tensor`` of shape ``(batch_size)``, `optional`, defaults to :obj: `None`):
             a one hot representation hof the correct answer `optional`
-        kwargs: (:obj:`Dict[str, any]`, optional, defaults to `{}`):
-            Used to hide legacy arguments that have been deprecated.
```

Comment:
Using kwargs results in more errors done in tests, and since it's the introduction of the model there's no need for legacy arguments management.